msteams-platform/apps-in-teams-meetings/in-meeting-notification-for-teams-meeting.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Build in-meeting notification for Teams meeting
+author: v-sdhakshina
+description: In this article, learn how to build in-meeting notification for Microsoft Teams meeting.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+---
+
+# Build in-meeting notification for Teams meeting
```

Comment:
@v-sdhakshina lets just be consistent over all to say "meeting" instead of "Teams meeting" in titles/ links.